packages/thirdweb/src/pay/buyWithFiat/getQuote.ts

```diff
@@ -0,0 +1,230 @@
+import type { ThirdwebClient } from "../../client/client.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import { getPayBuyWithFiatQuoteEndpoint } from "../utils/definitions.js";
+
+/**
+ * Parameters accepted by the [`getBuyWithFiatQuote`](https://portal.thirdweb.com/references/typescript/v5/getBuyWithFiatQuote) function for getting a quote to buy a token with fiat currency.
+ */
+export type GetBuyWithFiatQuoteParams = {
+  /**
+   * A client is the entry point to the thirdweb SDK. It is required for all other actions.
+   *
+   * You can create a client using the `createThirdwebClient` function.
+   * Refer to the [Creating a Client](https://portal.thirdweb.com/typescript/v5/client) documentation for more information.
+   */
+  client: ThirdwebClient;
+
+  /**
+   * The address of the wallet to which the tokens will be sent.
+   */
+  toAddress: string;
+
+  /**
+   * Chain id of the token to buy.
+   */
+  toChainId: number;
+
+  /**
+   * Token address of the token to buy.
+   */
+  toTokenAddress: string;
+
+  /**
+   * Symbol of the fiat currency to buy the token with.
+   *
+   * For "STRIPE" provider - only "USD" is supported.
```

Comment:
avoid mentions to the underlying provider - we might change these under the hood - details should not be exposed

Reply:
removed